openlibrary/templates/books/custom_carousel.html

```diff
@@ -30,19 +30,22 @@
     $ author_names = ''
   $ modifier = ''
   $ work_key = book.key if book.key.startswith('/work') else book.get('work_key')
-  $if lazy:
-    $ img_attr = 'data-lazy'
-  $else:
-    $ img_attr = 'src'
 
   <div class="book carousel__item">
     <div class="book-cover">
       <a href="$(url)">
         $if cover_url:
-          <img class="bookcover" loading="lazy"
-            width="130" height="200"
-            title="$('%s%s'%(title,byline))"
-            $img_attr="$(cover_url)"/>
+          $if lazy:
```

Comment:
This is fine, but there's no need to wrap the entire if statement as a lot of it is common code.

The following should also work.. can you confirm?:
```
<img class="bookcover" loading="lazy"
              width="130" height="200"
              title="$('%s%s'%(title,byline))"
$if lazy:
              data-lazy="$(cover_url)"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII="/>
$else:
              src="$(cover_url)"/>

Reply:
It's working 🎉

Reply:
This looks great to me!

I will pull it down for testing either Thursday or Friday in my next round of reviews.